scripts/windows-install-builder.nsi

```diff
@@ -68,12 +69,12 @@ RequestExecutionLevel user
 !define MUI_WELCOMEFINISHPAGE_BITMAP ".\Installer-Files\WelcomeFinishPage.bmp"
 !define MUI_UNWELCOMEFINISHPAGE_BITMAP ".\Installer-Files\WelcomeFinishPage.bmp"
 
-LangString WELCOME_TITLE ${LANG_INDONESIAN} "Indonesian title"
+LangString WELCOME_TITLE ${LANG_INDONESIAN} "Instalatur InaSAFE [[VERSION]]"
 LangString WELCOME_TITLE ${LANG_ENGLISH} "InaSAFE [[VERSION]] Installer"
 !define MUI_WELCOMEPAGE_TITLE $(WELCOME_TITLE)
 
-LangString WELCOME_TEXT ${LANG_INDONESIAN} "FOO BAR"
-LangString WELCOME_TEXT ${LANG_ENGLISH} "This setup tool will guide you through the process of installing the InaSAFE plugin. Note that it does not install QGIS itself - you must do that separately."
+LangString WELCOME_TEXT ${LANG_INDONESIAN} "Perangkat ini akan memandu Anda "
+LangString WELCOME_TEXT ${LANG_ENGLISH} "Alat pengaturan ini akan memandu Anda melalui proses menginstal plugin InaSAFE. Perhatikan bahwa itu tidak menginstal QGIS sendiri - Anda harus melakukannya secara terpisah."
```

Comment:
This doesn't look like english readable ...

Reply:
Ah thanks @Gustry 
I missed it, trying to copy and paste, but got it wrong. I will update soon.